cmd/agent/subcommands/flare/command.go

```diff
@@ -336,6 +342,20 @@ func makeFlare(flareComp flare.Component,
 		return err
 	}
 
+	streamLogParams := streamlogs.CliParams{
+		FilePath: commonpath.DefaultStreamlogsLogFile,
+		Duration: 60 * time.Second, // default duration
+		Quiet:    true,
+	}
+
+	if cliParams.withStreamLog {
+		fmt.Fprintln(color.Output, color.GreenString("Asking the agent to log the log-stream."))
```

Comment:
NIT: Perhaps something like:
```suggestion
		fmt.Fprintln(color.Output, color.GreenString("Asking the agent to stream logs"))
```